### PR TITLE
Update rule for ancestry.com in password-rules.json

### PR DESCRIPTION
**Pull request by 4amVim**
### Overall Checklist
- [x] I agree to the project's [Developer Certificate of Origin](https://github.com/apple/password-manager-resources/blob/main/DEVELOPER_CERTIFICATE_OF_ORIGIN.md)
- [x] The top-level JSON objects are sorted alphabetically
- [x] There are no [open pull requests](https://github.com/apple/password-manager-resources/pulls) for the same update

#### for password-rules.json
- [x] The given rule isn't particularly standard and obvious for password managers
- [x] Generated passwords have been tested from this rule using the [Password Rules Validation Tool](https://developer.apple.com/password-rules/)
- [x] Information has been included about the website's requirements (eg. screenshots, error messages, steps during experimentation, etc.)
- [x] The PR isn't documenting something that would be a common practice among password managers (e.g. minimal length of 6)

### CODE REVIEW

**Comment by Sameesunkaria** [comment]
@4amVim Can you please provide a link or screenshot from ancestry.com with the new rules?

**Comment by 4amVim** [comment]
Here's a screenshot showing that `byqwigvyktepTijza0`, a password generated by the proposed rule (`passwordrules="minlength: 8; required: digit; required: upper,lower;"`) indeed passes validation:


image

**Comment by Sameesunkaria** [comment]
Screenshot 2023-06-24 at 2 26 22 PM

File: quirks/password-rules.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
         "password-rules": "minlength: 6; maxlength: 15; allowed: lower, upper, digit;"
     },
     "ancestry.com": {
-        "password-rules": "minlength: 8; required: lower; required: upper; required: digit; required: [-!\"#$%&'()*+,./:;<=>?@[^_`{|}~]];"
+        "password-rules": "minlength: 8; required: lower,upper; required: digit;"
```
Comment (Sameesunkaria) [inline quirks/password-rules.json:48]:
```suggestion
        "password-rules": "minlength: 8; required: lower, upper; required: digit;"
```

Comment (Sameesunkaria) [reply quirks/password-rules.json:48]: The changes look good, just a minor whitespace issue.

**Review by Sameesunkaria — APPROVED** [review]

**Comment by Sameesunkaria** [comment]
Thank you for your contribution @4amVim :)

**Comment by 4amVim** [comment]
Thank you guiding me through it @Sameesunkaria :D

**Pull request merged**